### PR TITLE
Registration improvements

### PR DESCRIPTION
**Pull request by m-amparo-dev**

### CODE REVIEW

File: client/src/components/registration/Registration.jsx [hunk 1]
```diff
@@ -2,6 +2,10 @@ import React, { useState } from "react";
 import { useNavigate } from "react-router-dom";
 import "./Registration.css";
 import logo from "../../assets/images/village-logo.png";
+import Footer from "../footer/Footer.jsx";
```
Comment (catconqs) [inline client/src/components/registration/Registration.jsx:5]: Are you able to add in the NavComp (nav bar) on next PR?

File: client/src/components/registration/Registration.jsx [hunk 2]
```diff
@@ -28,7 +34,7 @@ const Registration = () => {
       return false;
     } else if (!passwordRegex.test(password)) {
       setPasswordError(
-        "Password must be at least 8 characters long and include 1 uppercase letter, 1 lowercase letter, 1 number and 1 special character (@$!%*?&#)"
+        "Password must have min. 8 characters, 1 uppercase and 1 lowercase letters, 1 number and 1 special character (@$!%*?&#)"
```
Comment (catconqs) [inline client/src/components/registration/Registration.jsx:37]: I'd change to:  "Password must have min. 8 characters, 1 uppercase letter and 1 lowercase letter, 1 number and 1 special character (@$!%*?&#)"

**Review by catconqs — APPROVED** [review]
- please update readme to request: npm install and update .env with myJWT etc
- it works so well!

File: client/src/components/registration/Registration.jsx (hunk 2 above)
Comment (m-amparo-dev) [reply client/src/components/registration/Registration.jsx:37]: Many thanks, I will update this.

**Comment by m-amparo-dev** [comment]
Many thanks, Cat!

**Pull request merged**